packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -169,13 +183,30 @@ class ObjectInspector extends Component {
     if (expand === true) {
       const {
         getObjectProperties,
+        getObjectEntries,
         loadObjectProperties,
+        loadObjectEntries,
       } = this.props;
 
       const value = getValue(item);
+      const parent = getParent(item);
+      const parentValue = getValue(parent);
+      const parentActor = parentValue
+        ? parentValue.actor
+        : null;
+
       if (nodeHasProperties(item) && value && !getObjectProperties(value.actor)) {
         loadObjectProperties(value);
       }
+
+      if (
+        nodeIsEntries(item)
```

Comment:
perhaps: `nodeHasEntries`

I could also see moving this to a util: `shouldLoadEntries`